index.html

```diff
@@ -8,6 +8,8 @@
     <meta charset="utf-8">
     <title>New Tab</title>
     <link rel="stylesheet" href="css/newTab.css">
+    <link rel="stylesheet" href="css/contentSearchUI.css">
+    <link rel="preconnect" id="preconnector" crossorigin="anonymous">
```

Comment:
I wonder if an engine should manage its own preconnector. That is, when the default engine is set, we create this element and add it to the DOM. Then, if the default engine is clobbered, it removes this from the DOM.